src/main/java/seedu/address/logic/commands/CreateStudentCommand.java

```diff
@@ -21,29 +17,36 @@
 public class CreateStudentCommand extends Command {
 
     public static final String COMMAND_WORD = "touch";
-    public static final String MESSAGE_USAGE = COMMAND_WORD + " : Adds a student into the specified directory\n"
-            + "Parameters: "
-            + "specified path "
-            + "[" + OPTION_NAME + " NAME] "
-            + "[" + OPTION_EMAIL + " EMAIL] "
-            + "[" + OPTION_PHONE + " PHONE] "
-            + "[" + OPTION_ADDRESS + " ADDRESS] "
-            + "Example: " + COMMAND_WORD
-            + " stu-200 "
-            + OPTION_NAME + " Bob "
-            + OPTION_EMAIL + " bobby@example.com "
-            + OPTION_PHONE + " 92929292 "
-            + OPTION_ADDRESS + " blk 258 Toa Payoh ";
+
+    public static final String MESSAGE_USAGE =
+            "Usage: " + COMMAND_WORD + " <path> " + "-n <value> " + "-e <value> " + "-p <value> " + "-a <value> \n"
+            + "\n"
+            + "Add new student.\n"
+            + "\n"
+            + "Arguments: \n"
+            + "    path                 Valid path to student\n"
+            + "    -n, --name           Name of the student\n"
+            + "    -e, --email          Email of the student\n"
+            + "    -p, --phone          Phone of the student\n"
+            + "    -a, --address        Address of the student\n"
+            + "\n"
+            + "Examples: \n"
+            + "touch grp-001/0001Y -n Mary -e mary@gmail.com -p 87652345 -a 4 Loyang Walk Loyang Industrial Estate";
```

Comment:
consider changing <value> to corresponding label, and also only -n is compulsory other opions are optional
e.g. -n <name>, -e <email> ...